rust/exchange_rate/src/declarations/exchange_rate/index.js

```diff
@@ -5,7 +5,7 @@ import { idlFactory } from "./exchange_rate.did.js";
 export { idlFactory } from "./exchange_rate.did.js";
 
 // CANISTER_ID is replaced by webpack based on node environment
-export const canisterId = process.env.EXCHANGE_RATE_CANISTER_ID;
+export const canisterId = process.env.CANISTER_ID_exchange_rate;
```

Comment:
since we migrate to vite, I think we should also use environment variable as when you use vite - i.e. `.env` and not `process.env...` but `import.meta.env...`

https://vitejs.dev/guide/env-and-mode.html

Reply:
I'm still not deprecating webpack at this point